resources/grub.rb

```diff
@@ -0,0 +1,31 @@
+#  Resource to manage the Grub configuration file
+#
+provides :grub
+
+property :path, String, name_property: true
+property :settings, Hash, default: {}
+
+def stringify
+  grub_config = ''
+  s = settings
```

Comment:
I don't like this and I suspect it'll throw errors/warnings. Should be `new_resource.settings`; or make it a parameter to `stringify`.